acceptance/support-skeletons/synced_folder_smb/Vagrantfile

```diff
@@ -0,0 +1,8 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "box"
+
+  config.vm.synced_folder ".", "/vagrant-smb", type: "smb", smb_username: ENV["SMB_USERNAME"], smb_password: ENV["SMB_PASSWORD"]
+  config.vm.synced_folder ".", "/foo", type: "smb", disabled: true
+  # Guest should persist mounts after reboot
+  config.vm.provision "shell", inline: "echo goodbye", reboot: true
```

Comment:
Maybe we want to set this provisioner to `run: "never"` and run it explicitly as part of the tests? That way we can check both of these scenarios:

1. Synced folder is mounted on the initial up
2. Synced folder persists after a reboot